src/Map/currentLocation.js

```diff
@@ -0,0 +1,151 @@
+/*global google*/
+import React from "react"
+import { compose, withProps, withHandlers, withState } from "recompose"
+import { withScriptjs, withGoogleMap, GoogleMap, Marker, InfoWindow, SearchBox} from "react-google-maps";
+import { Route, Link } from 'react-router-dom';
+import Geocode from 'react-geocode';
+
+import './map.css';
+
+
+const MapComponent = compose(
+    withProps({
+        googleMapURL: "https://maps.googleapis.com/maps/api/js?key=AIzaSyCTwzWuq9ff9eVO8s8rjpsIZef5fiABHPg&v=3.exp&libraries=geometry,drawing,places",
```

Comment:
This is showing the API key in the clear while below it's accessed from the environment variable.  I wondering if it might be better to have a common access pattern.

Reply:
That was the old free API key we were using. It's not being used anymore.

Reply:
Got it, this code is currently checked in, I guess that this component is not actually being used.  Is that right?

Reply:
The file you are referencing isn't being used anymore. I could remove it now.

Reply:
Yes exactly